config/locales/doorkeeper.en.yml

```diff
@@ -0,0 +1,123 @@
+en:
```

Comment:
@robguthrie can we exclude this from Transifex? I think it will confuse the living eff out of most of our translators :)